tests/XRootD/cache.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+export XrdSecPROTOCOL=host
+
+function setup_cache() {
+	require_commands diff
+	mkdir "${NAME}/xrootd/cache"
```

Comment:
You don't need the mkdir command, it's already done here:

https://github.com/xrootd/xrootd/blob/6579e066a25401ae6e6e3c235d494ef7884fb173/tests/XRootD/test.sh#L119

Reply:
Agreed.  Removed in the latest commit.